hazelcast/src/main/java/com/hazelcast/multimap/MultiMap.java

```diff
@@ -111,7 +111,9 @@
      * @param key   the key to be stored
      * @param value the value to be stored
      * @return {@code true} if size of the multimap is increased,
-     * {@code false} if the multimap already contains the key-value pair
+     * {@code false} if the collection type is SET and multimap already
+     * contains the key-value pair, {@code false} otherwise if
+     * the collection type is LIST
```

Comment:
For `LIST` type, method `put` returns true. To reword concisely we can have something like this:

```
{@code false} if the multimap already contained the key-value pair and doesn't allow duplicates
      
```